app/helpers/friendships_helper.rb

```diff
@@ -0,0 +1,13 @@
+module FriendshipsHelper
+    def requested(user_id)
```

Comment:
these methods should be on scope level please move them to the appropriate scopes